CHANGELOG.md

```diff
@@ -17,9 +17,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
  * When Intel RDT feature is not available, its initialization is skipped,
    resulting in slightly faster `runc exec` and `runc run`. (#3306)
- * Use `anon` + `file` from `memory.stat` for cgroupv2 root usage, as the root
-   does not have `memory.current` for cgroupv2. This aligns cgroupv2 root
-   usage more closely with cgroupv1 reporting. (#3933)
+ * Use `anon` + `file + swapcached` from `memory.stat` for cgroupv2 root
```

Comment:
nit: this should probably be `anon` + `file` + `swapcached`.

Reply:
fixed - although I'm not sure it's actually swapcached we want here? what do you think?